changelogs/fragments/add_ldap_search_module.yml

```diff
@@ -0,0 +1,2 @@
+feature:
```

Comment:
That's not valid syntax for changelog fragments (https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to).

Also, new modules should not have a changelog fragment at all. Please remove the changelog fragment.

(@gundalow I'll create a label 'new plugin' to track such PRs, so we'll know what new stuff was merged.)